templates/includes/sidebar.html

```diff
@@ -1,3 +1,11 @@
+{% if user.is_authenticated %}
+    User: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a>
+  {% else %}
+    <a href="{% url 'login' %}">Login</a>
+  {% endif %}
+<br>
+<a href="{% url 'logout' %}">Logout</a>
```

Comment:
The 'Logout' link is always displayed, regardless of the user's authentication status. Consider moving this link inside the `{% if user.is_authenticated %}` block to ensure it only appears when a user is logged in.